### PR TITLE
increase prettier tab width to 4 for proper documentation rendering

### PR DESCRIPTION
**Pull request by pmeier**
This PR looks massive, but it is mostly automatic whitespace changes. The only manual change is in `.prettierrc`. I'll flag it below.

### CODE REVIEW

File: .prettierrc.yaml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 proseWrap: always
 printWidth: 88
+tabWidth: 4
```
Comment (pmeier) [inline .prettierrc.yaml:3]: Flag.

File: docs/index.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ title: Home
 
 # ![Ragna logo](assets/brand/logo-lockup-horizontal/logo-lockup-horizontal.png)
 
+- Here is a nested list with a tab width of 2
+  - First item
+  - Second item
+- Here is a nested list with a tab width of 4
+    - First item
+    - Second item
```
Comment (pmeier) [inline docs/index.md:12]: This is just here to showcase a tab width 2 vs 4. 15bb981ca271895f94adef0d16c3034e539382a3 needs to be reverted before merge.

Comment (pmeier) [reply docs/index.md:12]: Here is the rendered result

image

Using only 2 tabs flattens the sublist.

**Comment by pmeier** [comment]
Before this gets approved / merged, please have a look at #324 that enables sublists with a tab width of 2 in `mkdocs` with an extension. I'm ok with both solutions, but would prefer #324. Feel free to approve one of the PRs and close the other.

**Comment by pavithraes** [comment]
Thanks! Closing in favor of #324 :)

**Pull request closed without merging**